ci-operator/step-registry/ovn/conf/custom-cidr/ovn-conf-custom-cidr-ref.yaml

```diff
@@ -11,5 +11,9 @@ ref:
     default: "10.128.0.0/20"
   - name: CLUSTER_NETWORK_HOST_PREFIX
     default: "23"
+  - name: CLUSTER_NETWORK_SEC_CIDR
+    default: "no"       
+  - name: CLUSTER_NETWORK_SEC_HOST_PREFIX
+    default: "no"
   documentation: |-
-   this will deploy cluster with custom the cidr to define the cluster workers numbers
+   this will deploy cluster with custom the cidr to define the cluster workers numbers, CLUSTER_NETWORK_SEC_CIDR for second cluster subnet and CLUSTER_NETWORK_SEC_HOST_PREFIX for second hostprefix    
```

Comment:
cluster workers number?

Reply:
yes,  the hostprefix and host subnet can decide the worker numbers.